src/preamble.js

```diff
@@ -1313,11 +1313,19 @@ try {
 var TOTAL_STACK = Module['TOTAL_STACK'] || {{{ TOTAL_STACK }}};
 var TOTAL_MEMORY = Module['TOTAL_MEMORY'] || {{{ TOTAL_MEMORY }}};
 var FAST_MEMORY = Module['FAST_MEMORY'] || {{{ FAST_MEMORY }}};
+#if POINTER_MASKING
+var POINTER_MASKING = Module['POINTER_MASKING'] || {{{ POINTER_MASKING }}};
```

Comment:
why are these variables variable? is there a situation where they would be modified? if not, you can access them using `{{{ X }}}` wherever they are used.